### DISCUSSION TITLE
Handle alarms independently?

### DISCUSSION THREAD

**Discussion by derandiunddasbo** [DurgNomis-drol/google_home_timers_card#17] [Ideas]
Currently it's possible to create a card showing only active timers, but it's not possible to have a card showing only active alarms for a GH device. Is there a (technical) reason for this limitation / dependency?

**Comment by ThibautLM** [comment]
I was looking for the same thing ! Is it possible to add an option like : show_inactive_alarms = false to hide inactive alarme based on his status (set/inactive) ? Each alarm_id has a status ... 
Nice work in general ! 

Thanks in advance

**Reply by DurgNomis-drol** [reply]
No it is not. The card is not maintained any more. See notice at the top of the README

> Nice work in general !

Thanks!
